panel/template/fast/grid/fast_grid_template.css

```diff
@@ -187,7 +187,7 @@ html {
   scrollbar-shadow-color: var( --neutral-fill-rest);
 }
 
-::-webkit-scrollbar { width: 12px; height: 3px;}
```

Comment:
Is there a reason why this definition isn't in `panel/template/fast/css/fast_root.css`?

Reply:
Legacy reasons and laziness :-)

I've just pushed a change to move it to fast_root.css.